lib/trusted-apps.js

```diff
@@ -90,6 +93,29 @@ function middleware(req, res, next) {
   next();
 }
 
+function middlewareForKoa(ctx) {
```

Comment:
This middleware never seems to call next() as it should. I think it happens to work where you're using it in your application because you're not using it like middleware, you're just calling it as function inside some other request handler, but if it was used as middleware I think the request would never complete.

https://github.com/koajs/koa#middleware

Basically update the arguments to `(ctx, next)` and then `return next();` as the last line in the function.